components/application-broker/internal/broker/testing/listers.go

```diff
@@ -0,0 +1,57 @@
+package testing
+
+import (
+	fakeistioclientset "istio.io/client-go/pkg/clientset/versioned/fake"
+	"k8s.io/apimachinery/pkg/runtime"
+	fakekubeclientset "k8s.io/client-go/kubernetes/fake"
+	"k8s.io/client-go/tools/cache"
+	fakeeventingclientset "knative.dev/eventing/pkg/client/clientset/versioned/fake"
+	"knative.dev/pkg/reconciler/testing"
+)
+
+var clientSetSchemes = []func(*runtime.Scheme) error{
+	fakekubeclientset.AddToScheme,
+	fakeeventingclientset.AddToScheme,
+	fakeistioclientset.AddToScheme,
+}
+
+type Listers struct {
+	sorter testing.ObjectSorter
+}
+
+func NewScheme() *runtime.Scheme {
+	scheme := runtime.NewScheme()
+
+	for _, addTo := range clientSetSchemes {
+		addTo(scheme)
```

Comment:
you should handle the error from underlying method `func (sb *SchemeBuilder) AddToScheme(s *Scheme) error` here.
```suggestion
		if err := addTo(scheme); err != nil {
		return nil, errors.Wrapf(err, "while adding to scheme")
		}
```